integration/client/client_test.go

```diff
@@ -380,7 +380,7 @@ func TestImagePullSomePlatforms(t *testing.T) {
 
 	// Note: Must be different to the image used in TestImagePullAllPlatforms
 	// or it will see the content pulled by that, and fail.
-	img, err := client.Fetch(ctx, "k8s.gcr.io/pause:3.2", opts...)
+	img, err := client.Fetch(ctx, "k8s.gcr.io/busybox:1.29-2", opts...)
```

Comment:
the pause update looks to be complete, but this image reference looks strange; if I look at all the busybox tags, the only thing somewhat like this is `1.27.2`:

```
$ curl -s https://k8s.gcr.io/v2/busybox/tags/list | jq .tags
[
  "1.24",
  "1.27",
  "1.27.2",
  "latest"
]
```

Reply:
Indeed, but the tag should be 1.29-2 from what I see here from the k8s image promoter: https://github.com/kubernetes/k8s.io/blob/main/k8s.gcr.io/images/k8s-staging-e2e-test-images/images.yaml#L24

Reply:
cool, yeah, that one looks right--multi-arch with 9 platform entries

Reply:
yeah this one or the downlevel pause would also be fine..

Reply:
> yeah this one or the downlevel pause would also be fine..

Not for Windows. Only latest tags have WS2022 in manifest lists.

Reply:
yeah .. that test was just making sure a subset could be retrieved vs .. the all test above it